pom.xml

```diff
@@ -58,6 +58,24 @@
                 <artifactId>jackson-databind</artifactId>
                 <version>${jackson.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.mockito</groupId>
+                <artifactId>mockito-core</artifactId>
+                <version>4.5.1</version>
```

Comment:
would be nice to see this as property like mockito-core.version or similar but for now it's ok to stay like this